.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
This is an interesting framing that we may want to change, having the first example of a feature request being "I'm always frustated when ..." doesn't necessarily set a great tone for bug reports. I know it drains a lot out of me at least reading issues that are just frustrated rants.